index.html

```diff
@@ -11821,7 +11821,8 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>If the interface is such that it is not possible to have a visible label on the screen, authors SHOULD use <pref>aria-label</pref> and SHOULD NOT use <pref>aria-labelledby</pref>. Authors MUST NOT specify <code>aria-labelledby</code> on an element which has an explicit or implicit WAI-ARIA role where <code>aria-labelledby</code> is <a href="#prohibitedattributes">prohibited</a>. As required by the <a href="#textalternativecomputation">accessible name and description computation</a>, user agents give precedence to <pref>aria-labelledby</pref> over <pref>aria-label</pref> when computing the accessible name property.</p>
 				<p>The <pref>aria-labelledby</pref> attribute is similar to <pref>aria-describedby</pref> in that both reference other elements to calculate a text alternative (an accessible name, and description, respectively). While a concise accessible name is preferable, a description can either be concise, or provide more verbose information.</p>
 				<!-- keep previous sentence synced with the associated description in #aria-describedby -->
-				<p class="note">The expected spelling of this property in <abbr title="United States">U.S.</abbr> English is "labeledby." However, the <a>accessibility <abbr title="Application Programing Interfaces">API</abbr></a> features to which this property is mapped have established the "labelledby" spelling. This property is spelled that way to match the convention and minimize the difficulty for developers.</p>
+				<p class="note">The expected spelling of this property in <abbr title="United States">U.S.</abbr> English is "labeledby." However, the <a>accessibility <abbr title="Application Programing Interfaces">API</abbr></a> features to which this property is mapped have established the "labelledby" spelling.</p>
```

Comment:
```suggestion
				<p class="note">The expected spelling of this property in <abbr title="United States">U.S.</abbr> English is "labeled by." However, the <a>accessibility <abbr title="Application Programing Interfaces">API</abbr></a> feature this property was originally based on map had established the <abbr title="United Kingdom">U.K.</abbr> English spelling, "labelled by."</p>
```

Reply:
Away form my computer right now, so making this note to accept this suggestion, but fix this text:
“was originally based on **map had** established”